<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>322</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>291</width>
     <height>291</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>10</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>VGG16 TEST</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ShowTrainImageButton">
      <property name="text">
       <string>1.show Train Image</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ShowHyperParameterButton">
      <property name="text">
       <string>1.show HyperParameter</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ShowModelShortCutButton">
      <property name="text">
       <string>3. Show Model Shortcut</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="ShowAccButton">
      <property name="text">
       <string>4.show Accuracy</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="TestIndexInput"/>
    </item>
    <item>
     <widget class="QPushButton" name="TestButton">
      <property name="text">
       <string>5.Test</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
